Roles/(Ghosts)/Impostor/Bloodmoon.cs

```diff
@@ -94,12 +93,13 @@ public override void OnOtherTargetsReducedToAtoms(PlayerControl target)
         if (LastTime.ContainsKey(targetid))
             LastTime.Remove(targetid);
     }
-    private string OthersNameText(PlayerControl seer, PlayerControl player, bool IsForMeeting) 
+    private string OthersNameText(PlayerControl seer, PlayerControl player, bool IsForMeeting, bool isforhud = false) 
     {
-        if (IsForMeeting || seer != player) return string.Empty;
+        var IsMeeting = GameStates.IsMeeting;
+        if (IsMeeting || seer != player || player == null || isforhud) return string.Empty;
         var playerid = player.PlayerId;  
 
-        if (LastTime.TryGetValue(playerid, out var lastTime) && lastTime + 1 <= GetTimeStamp() && !IsForMeeting)
+        if (LastTime.TryGetValue(playerid, out var lastTime) && lastTime + 1 <= GetTimeStamp() && !IsMeeting)
```

Comment:
text in `isForHud` will not be displayed for modded players

And `GameStates.IsMeeting` doesn't work correctly in NotifyRoles, so better use `IsForMeeting`

Reply:
When I used IsForMeeting, the target got killed during meeting..

I think it's maybe bcuz it returns false here? 
![image](https://github.com/0xDrMoe/TownofHost-Enhanced/assets/143632280/437c5c2d-c585-4281-8955-dd9636e209bc)

Reply:
Then use `isForMeeting || GameStates.IsMeeting` and return `string.Empty`

Reply:
done